ui/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -84,7 +84,7 @@ function handleError(context: IActionContext, callbackId: string, error: unknown
 
     if (!context.errorHandling.suppressDisplay) {
         // Always append the error to the output channel, but only 'show' the output channel for multiline errors
-        ext.outputChannel.appendLine(localize('outputError', 'Error: {0}', errorData.message));
+        ext.outputChannel.appendLog(localize('outputError', 'Error: {0}', errorData.message));
```

Comment:
I think `appendLog` makes complete sense for long-running operations like the wizard steps, but I'm not sure we want to do `appendLog` for _everything_. If you want to do that, I'd like to see some screenshots and get Matt's opinion first.

Reply:
![image](https://user-images.githubusercontent.com/5290572/59539162-0d007480-8eb1-11e9-893a-509b5bd18d10.png)

Note: `appservice`  package isn't updated yet thus no timestamps